scripts/roles/todd/tasks/main.yml

```diff
@@ -0,0 +1,123 @@
+---
+- include_vars: toddvars.yml
+
+- name: Set PATH and GOPATH and for Vagrant user and shell profiles
+  lineinfile:
+    dest: "{{ item.file }}"
+    state: "present"
+    regexp: "^export {{ item.varname }}"
+    line: "{{ item.varval }}"
+  with_items:

```

Comment:
Nice! Looks a lot cleaner. Done in f028bf8

Also, both `gopath` and `path` are defined further up in `vars/main.yml`